dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOptionTest.java

```diff
@@ -16,72 +16,41 @@
 
 package com.google.cloud.tools.opensource.classpath;
 
-import com.google.common.collect.ImmutableList;
-import com.google.common.truth.Truth;
+import org.apache.commons.cli.CommandLine;
 import org.apache.commons.cli.ParseException;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class StaticLinkageCheckOptionTest {
-  @Test
-  public void parseCommandLineOptions_shortOptions_bom() throws ParseException {
-    String[] arguments = new String[] {
-        "-b", "abc.com:dummy:1.2",
-        "-r"
-    };
-    StaticLinkageCheckOption parsedOption = StaticLinkageCheckOption.parseArguments(arguments);
-
-    Assert.assertEquals("abc.com:dummy:1.2", parsedOption.getBom());
-    Truth.assertThat(parsedOption.getArtifacts()).isEmpty();
-    Truth.assertThat(parsedOption.getJarFiles()).isEmpty();
-    Assert.assertTrue(parsedOption.isReportOnlyReachable());
-  }
 
   @Test
-  public void parseCommandLineOptions_longOptions() throws ParseException {
-    String[] arguments = new String[] {
-        "--artifacts", "abc.com:abc:1.1,abc.com:abc-util:1.2",
-        "--report-only-reachable"
-    };
-    StaticLinkageCheckOption parsedOption = StaticLinkageCheckOption.parseArguments(arguments);
+  public void parseCommandLineOptions_shortOptions_bom() throws ParseException {
+    String[] arguments = {"-b", "abc.com:dummy:1.2", "-r"};
+    CommandLine parsedOption = StaticLinkageCheckOption.readCommandLine(arguments);
 
-    Assert.assertEquals(ImmutableList.of("abc.com:abc:1.1", "abc.com:abc-util:1.2"),
-        parsedOption.getArtifacts());
-    Assert.assertTrue(parsedOption.isReportOnlyReachable());
+    Assert.assertEquals("abc.com:dummy:1.2", parsedOption.getOptionObject('b'));
   }
 
   @Test
   public void parseCommandLineOptions_duplicates() throws ParseException {
```

Comment:
This function does not seem to throw ParseException any more.

Reply:
done